src/api/public/apidocs-new/paths/source_project_name_attribute_attribute_name.yaml

```diff
@@ -5,19 +5,26 @@ get:
   parameters:
     - $ref: '../components/parameters/project_name.yaml'
     - $ref: '../components/parameters/attribute_name.yaml'
-    - name: meta
-      in: query
-      schema:
-        type: string
-    - name: rev
+    # Query Strings:
+    # `rev`: doesn't make sens in project level and makes the endpoint crash.
+    # `meta`: either you pass `meta` or not, it's always replaced by meta: 1.
+    # `with_project` does not make sense in project level and makes the endpoint crash.
```

Comment:
In [this line](https://github.com/openSUSE/open-build-service/blob/4612bdca783e6cfc5c240ca3d063c98bcc4cbbc5/src/api/app/mixins/has_attributes.rb#L71) it's not possible to access to `project` is `self` is a project already.